### DISCUSSION TITLE
Bookmarks Widget - Add Name Option to Order By

### DISCUSSION THREAD

**Discussion by Chankster** [netbox-community/netbox#16084] [Ideas]
In the bookmarks widget on the dashboard, it would be nice if there was an option to order by "Name" rather than just "Newest/Oldest"

**Comment by jeremystretch** [comment]
I'm afraid this is not feasible as not all models have a `name` attribute.

**Comment by Chankster** [comment]
Closing discussion as this was implemented: https://github.com/netbox-community/netbox/pull/16426
